src/main/java/core/basesyntax/model/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.model;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double topBase;
+    private double bottomBase;
+    private double height;
+
+    public IsoscelesTrapezoid(double topBase, double bottomBase, double height, Color color) {
+        super(color);
+        this.topBase = topBase;
+        this.bottomBase = bottomBase;
+        this.height = height;
+    }
+
+    @Override
+    public double calculateArea() {
+        double area = 0.5 * height * (topBase + bottomBase);
+        return Math.round(area * 10.0) / 10.0;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println(super.getClass().getSuperclass().getName()
+                + ": isoscelesTrapezoid, "
+                + "area: " + calculateArea() + " sq.units, "
+                + "topBase: " + topBase + " units"
+                + ", bottomBase: " + bottomBase + " units"
+                + ", height:" + height + " units"
+                + ", color: " + getColor());
```

Comment:
[Use name() for getting String representation of enum constants](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#use-name-for-getting-string-representation-of-enum-constants)